tests/core/Jalasoft.TeamUp.Projects.Core.Tests/ProjectsServiceTests.cs

```diff
@@ -88,7 +89,7 @@ public void GetProject_Returns_SingleProject()
         }
 
         [Fact]
-        public void GetProject_Throws_ProjectsException()
+        public void GetProject_IdIsNotValid_NotFound()
         {
             // Arrange
             this.mockRepository.Setup(repository => repository.GetById(Guid.Parse("4a7939fd-59de-44bd-a092-f5d8434584df"))).Throws(new ProjectsException(ProjectsErrors.NotFound, new Exception()));
```

Comment:
we shouldn't throw project exceptions at service level, please remove the reference of the utils project from service project